new-features.md

```diff
@@ -0,0 +1,20 @@
+---
+title: "Can the staff add new features?"

```

Comment:
Need examples of features.
1. Mods? We don't manage mods
2. Server cores? We can change server cores. Link to that FAQ page
3. Others? Need more examples